### PR TITLE
pushing provision file with latest script

### PR DESCRIPTION
**Pull request by roshnikmicrofocus**

### CODE REVIEW

File: scripts/Provision.ps1 [hunk 1]
```diff
@@ -34,10 +34,10 @@ try {
     Start-Service PlateSpin_Management_Service -ErrorAction SilentlyContinue
     Start-Service OfxController -ErrorAction SilentlyContinue
 
-    C:\Windows\OEM\ForgeApplianceConfigurator\ForgeApplianceConfigurator.exe /skip_network_config /cloud_config_only /hosting_cloud="aws" /alternate_address=$publicIp /log=$LogFile
+    C:\Windows\PlateSpin\ForgeApplianceConfigurator\ForgeApplianceConfigurator.exe /skip_network_config /cloud_config_only /hosting_cloud="aws" /alternate_address=$publicIp /log=$LogFile
```
Comment (mehendran) [inline scripts/Provision.ps1:37]: We should have kept PlateSpin Folder under C:\Windows\OEM. 

It is ok for this time.

File: templates/quickstart-platespin-master-template.yaml [hunk 2]
```diff
@@ -210,7 +210,7 @@ Parameters:
     ConstraintDescription: >-
       Quick Start bucket name can include numbers, lowercase letters, uppercase
       letters, and hyphens (-). It cannot start or end with a hyphen (-).
-    Default: aws-quickstart
+    Default: aws-quickstart-migrate
```
Comment (mehendran) [inline templates/quickstart-platespin-master-template.yaml]: We shouldn't change the default value.

File: templates/quickstart-platespin-template.yaml [hunk 3]
```diff
@@ -175,7 +175,7 @@ Parameters:
     ConstraintDescription: >-
       Quick Start bucket name can include numbers, lowercase letters, uppercase
       letters, and hyphens (-). It cannot start or end with a hyphen (-).
-    Default: aws-quickstart
+    Default: aws-quickstart-migrate
```
Comment (mehendran) [inline templates/quickstart-platespin-template.yaml]: We shouldn't change the default value.

File: templates/quickstart-platespin-master-template.yaml (hunk 2 above)
Comment (roshnikmicrofocus) [reply templates/quickstart-platespin-master-template.yaml]: changed it back to default

File: templates/quickstart-platespin-template.yaml (hunk 3 above)
Comment (roshnikmicrofocus) [reply templates/quickstart-platespin-template.yaml]: changed it back to default

**Review by mehendran — APPROVED** [review]

**Pull request merged**